src/components/form-elements/pages/select.jsx

```diff
@@ -57,6 +65,14 @@ const selects = () => (
     <DocsBlock info="Full width">
       <Select {...exampleProps} fullWidth />
     </DocsBlock>
+    <DocsBlock info="With grouped options">
+      <Select
+        {...exampleProps}
+        options={[]}
```

Comment:
I can see that `options` are optional, so we can remove this line not to confuse people how to use it properly.